### PR TITLE
Fix azure eventhubs pubsub certification test workflow

### PR DESCRIPTION
**Pull request by berndverst**
Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

# Description

Adds a missing secret to the EventHubs PubSub certification workflow

### CODE REVIEW

**Comment by berndverst** [comment]
cc @surenderssm 

@artursouza @yaron2 @mukundansundar or @daixiang0 please approve when you have a chance :)

**Review by mukundansundar — APPROVED** [review]

**Comment by surenderssm** [comment]
Thanks @berndverst for this catch and fix, looks good to me.

**Comment by daixiang0** [comment]
@berndverst thanks for your fix! Good job!

**Comment by daixiang0** [comment]
@artursouza sorry for merge :(

**Comment by berndverst** [comment]
> @artursouza sorry for merge :(

It's ok -- I would have merged it myself as a maintainer after your approval :)

**Pull request merged**
